src/config/featureFlags/schema.ts

```diff
@@ -100,6 +101,7 @@ export const mapFeatureFlagsEnvToState = (config: IFeatureFlags) => {
 
     showMarket: config.market,
     enableSTT: config.speech_to_text,
+    useModelName: config.model_name,
```

Comment:
key can be `modelTagUseModelName`, value is `config.model_tag_display_mode === 'model_name'`